zerver/webhooks/thinkst/doc.md

```diff
@@ -1,20 +1,32 @@
-See your Thinkst Canary and Canarytoken alerts in Zulip! This integration works with
-Canarytokens from Thinkst's paid product, not [canarytokens.org][canarytokens] - see the
-[Canarytokens](/integrations/doc/canarytoken) integration for those!
+# Zulip Thinkst integration
+
+Get your Thinkst Canary and Canarytoken alerts in Zulip!
+
+!!! warn ""
+    **Note**: This integration works with Canarytokens from Thinkst's paid product.
+    For [canarytokens.org][canarytokens], see the
+    [Canarytokens](/integrations/doc/canarytoken) integration!
+
+{start_tabs}
 
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
 1. Go to your Thinkst Canary settings, and click on **Webhooks** on
    the left sidebar. Select the **Generic** tab. Press
-   **Add Generic Webhook** and enter the constructed URL above. Finally,
-   click **Save**.
+   **Add Generic Webhook** and enter the generated URL above. Click **Save**.
+
+{end_tabs}
 
 {!congrats.md!}
 
 ![](/static/images/integrations/thinkst/001.png)
 
+### Related documentation
+
+{!webhooks-url-specification.md!}
+
 [canarytokens]: https://canarytokens.org
```

Comment:
For the new reference links that I've created in the other recent PRs, I have used just numbers, we could also use short names like these.